src/main/java/com/contrastsecurity/maven/plugin/ContrastInstallAgentMojo.java

```diff
@@ -21,41 +21,94 @@
 import org.apache.maven.plugins.annotations.Parameter;
 import org.apache.maven.project.MavenProject;
 
+/**
+ * Includes the Contrast Java agent in integration testing to provide Contrast Assess runtime
+ * security analysis
+ */
 @Mojo(name = "install", defaultPhase = LifecyclePhase.VALIDATE, requiresOnline = true)
 public final class ContrastInstallAgentMojo extends AbstractAssessMojo {
 
   @Parameter(defaultValue = "${project}", readonly = true)
   private MavenProject project;
 
+  /**
+   * When {@code true}, will not alter the Maven {@code argLine} property
+   *
+   * @since 2.0
+   */
   @Parameter(property = "skipArgLine")
   boolean skipArgLine;
 
+  /**
+   * "true" if this is a standalone application (e.g. uses an embedded web server vs war packaging)
+   *
+   * @since 2.2
+   */
   @Parameter(property = "standalone")
   boolean standalone;
 
-  @Parameter(property = "profile")
-  private String profile;
-
+  /**
+   * Override the reported server environment. {@see
+   * https://docs.contrastsecurity.com/en/server-configuration.html}
+   */
```

Comment:
Missing `@since 2.9`

Reply:
Ugh the README and the docs.contrastsecurity.com content are out of sync so I missed some of these. Good catch